### PR TITLE
Admin Endpoints

### PR DESCRIPTION
**Pull request by gerardo-rivera3901**
Created endpoints to get all users and get all rooms in order to access them within the admin settings. Needs middleware to verify that a user has an administrative role.

### CODE REVIEW

**Review by s-luong07 — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -69,6 +68,7 @@ app.use('/api/auth', authRouter);
 app.use('/api/user', userRouter);
 app.use('/api/post', verifyRole, postRouter);
 app.use('/api/comment', verifyRole, commentRouter);
+app.use('/api/admin', AdminRouter);
```
Comment (salza323) [inline index.js]: Should have the verifyRole middleware passed in to make sure that only admin(role3) can have access to these endpoints.

File: routes/admin.js [hunk 2]
```diff
@@ -1,8 +1,27 @@
-const express = require("express");
-const Admin = require("../models/admin");
+const express = require('express');
+const Admin = require('../models/admin');
 
 const app = express.Router();
 
+app.get('/users', async (req, res) => {
+  try {
```
Comment (salza323) [inline routes/admin.js]: Will need to check role_id here before the "get" is done

**Review by salza323 — DISMISSED** [review]
Gerardo, please see comments in the index and routes file. We just need to be sure to check the role_id before the user is granted access to any endpoints that are to be reserved to am admin. Examples of this can be found in the room-route file when creating and deleting a room.

**Review by jamesjlundin — APPROVED** [review]
added in the check that user is admin before returning data

**Pull request merged**